code/game/machinery/computer/security.dm

```diff
@@ -919,8 +922,7 @@ What a mess.*/
 					if("Change Rank")
 						if(active1)
 							active1.fields["rank"] = strip_html(href_list["rank"])
-							if(href_list["rank"] in SSjob.station_jobs)
-								active1.fields["real_rank"] = href_list["real_rank"]
```

Comment:
Was this param unused ?

Reply:
Yes, this is the only instance of real_name used in the entire codebase. Just a cleanup.